orchestrator/careplancontributor/applaunch/zorgplatform/service.go

```diff
@@ -377,6 +377,12 @@ func (s *Service) getSessionData(ctx context.Context, accessToken string, launch
 
 	serviceRequest := &fhir.ServiceRequest{
 		Status: fhir.RequestStatusActive,
+		Identifier: []fhir.Identifier{
+			{
+				System: to.Ptr("https://api.zorgplatform.online/fhir/v1/Task"),
```

Comment:
This system looks a lot like an actual URL of the Zorgplatform workflow API. Is that practical? Might cause confusion?

Reply:
It could, perhaps when using agains ZPF ACC. Used the value that was designed in the ticket, I think it should be okay as the system is "only" a namespace. It does clearly map to a workflow-id Task, a more generic value could be something like `https://zorgplatform.online/workflows`